client/reader/following/main.jsx

```diff
@@ -10,13 +10,25 @@ import SuggestionProvider from 'calypso/reader/search-stream/suggestion-provider
 import Stream, { WIDE_DISPLAY_CUTOFF } from 'calypso/reader/stream';
 import ReaderListFollowedSites from 'calypso/reader/stream/reader-list-followed-sites';
 import Recent from '../recent';
+import { useFollowingView } from './view-preference';
+import ViewToggle from './view-toggle';
 import './style.scss';
 
 function FollowingStream( { ...props } ) {
+	const { currentView, setView } = useFollowingView();
+
+	const handleViewChange = ( view ) => {
+		setView( view );
+	};
+
+	const viewToggle = config.isEnabled( 'reader/recent-feed-overhaul' ) ? (
+		<ViewToggle currentView={ currentView } onChange={ handleViewChange } />
+	) : null;
```

Comment:
Nit: We can do this in fewer lines of code.

```suggestion
	const viewToggle = config.isEnabled( 'reader/recent-feed-overhaul' ) ? (
		<ViewToggle currentView={ currentView } onChange={ ( view ) => setView( view ) } />
	) : null;
```

Reply:
Thanks for catching this! I left a redundant function in there. Updated in https://github.com/Automattic/wp-calypso/pull/96798/commits/7afcdd3049f33eca628601266c5f5ba73943c8ac